tck/src/main/java/org/eclipse/configjsr/configsources/ConfigurableConfigSource.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```

Comment:
2016-2018

Reply:
bulk replaced/fixed all Copyright years to 2018